WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewModelTests.swift

```diff
@@ -14,6 +14,127 @@ final class CustomerSelectorViewModelTests: XCTestCase {
         stores = MockStoresManager(sessionManager: .testingInstance)
     }
 
+    func test_isEligibleForAdvancedSearch_when_wc_plugin_version_is_lower_than_minimum_then_calls_action_with_right_parameters_and_return_false() {
+        // Given
+        var passedParameters: (Int64, String)?
+        let returnedVersion = "7.9.9"
+
+        let viewModel = CustomerSelectorViewModel(siteID: sampleSiteID, stores: stores) { _ in }
+
+        stores.whenReceivingAction(ofType: SystemStatusAction.self) { action in
+            switch action {
+            case let .fetchSystemPlugin(siteID, pluginName, onCompletion):
+                passedParameters = (siteID, pluginName)
+
+                onCompletion(SystemPlugin.fake().copy(version: returnedVersion))
+            default:
+                break
+            }
+        }
+
+        // When
+        var result = true
+        waitForExpectation { expectation in
+            viewModel.isEligibleForAdvancedSearch() { isEligible in
+                result = isEligible
+
+                expectation.fulfill()
+            }
+        }
+
+        // Then
+        XCTAssertFalse(result)
```

Comment:
Just for reading clarity, what do you think if we switch the variable names here (and the other tests)? 

```suggestion
        var isEligible = true
        waitForExpectation { expectation in
            viewModel.isEligibleForAdvancedSearch() { result in
                isEligible = result

                expectation.fulfill()
            }
        }

        // Then
        XCTAssertFalse(isEligible)
```

Reply:
Good idea! Done in adef247b2253f4a0818e4fc9af42b52d1bb69b45